src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -447,6 +447,10 @@ public async Task<uint256> SendManyAsync(string fromAccount, string addressesJso
 
                 return transaction.GetHash();
             }
+            catch (SecurityException exception)
```

Comment:
When do we throw this exception?

Reply:
``` Key privateKey = Key.Parse(wallet.EncryptedSeed, walletPassword, wallet.Network);```

Will throw it if invalid(or no password) is passed in.